### DISCUSSION TITLE
increase font size for cluster labels in umap

### DISCUSSION THREAD

**Discussion by sid5427** [GreenleafLab/ArchR#2012] [Questions / Documentation]
Hello,

I want to increase the size of the cluster labels which are generated in the UMAP key - I looked at ggpoint() but did not see specific option for that - could you please suggest how to do this?

Here is my snippet of code plotting the umap

```
p1 <- plotEmbedding(ArchRProj = proj_bigcs_2, 
colorBy = "cellColData", 
name = "clusters", 
embedding = "customUMAP",
size=0.5,
baseSize=20)
```

Thanks!

**Comment by drowsygoat** [comment]
Try 'labelSize'
